lib/enyo/lib/default-templates/default-app/.gitignore

```diff
@@ -1,2 +1,4 @@
 .enyocache
-dist/
\ No newline at end of file
+dist/
```

Comment:
We could remove it from all them frankly since none of the stuff by default uses node_modules.  Moreso about future usage I guess; like the jshint stuff